src/ImageSharp/Colors/Spaces/CieLab.cs

```diff
@@ -161,17 +161,17 @@ public override bool Equals(object obj)
         /// <inheritdoc/>
         public bool Equals(CieLab other)
         {
-            return this.AlmostEquals(other, Constants.Epsilon);
+            return this.backingVector.Equals(other.backingVector);
```

Comment:
Are you sure this change is correct? Maybe @JimBobSquarePants can comment if the `AlmostEquals` was intended behavior?